COVID-19 as an anthroponotic disease (#1141)
greenelab/covid19-review: content/09.evolution.md

A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of more thorough monitoring of zoonotic diseases.
More importantly, it would allow researchers to have a better understanding of and to community concerns about potential emerging viral threats.

### Question 3: Which Species Are Susceptible?

Given the strong evidence for a zoonotic origin of SARS-CoV-2, another evolutionary question that received significant attention, especially early on, was whether humans could infect other species with SARS-CoV-2.

> In the modern age, opportunities for human-to-animal transmission events could arise in interactions with companion animals, zoo animals, house pests, and livestock.

Reviewer: ```suggestion
In the modern age, opportunities for human-to-animal transmission events could arise in interactions with companion animals, zoo animals, house pests, hunting, urbanized wildlife, and livestock.
```

Author: OMG how could I forget urban wildlife, that was a huge part of my PhD :laughing:

Recognition of this protein is largely determined by domains in the S1 subunit of the RBD [@doi:10.1016/j.cell.2020.02.058].
Alignment of the _ACE2_ sequence from 19 species revealed high conservation among mammals [@doi:10.1016/j.jinf.2020.02.013].
This analysis suggested that non-human primates (three monkey and two ape species), companion animals (dogs and cats), and livestock (both odd- and even-toed ungulates) may all be susceptible to SARS-CoV-2 [@doi:10.1016/j.jinf.2020.02.013].
Similarly, another study conducted an _in silico_ analysis of ACE2 protein structures and their predicted binding to SARS-CoV-2 for 410 vertebrate species [@doi:10.1073/pnas.2010146117].
The species identified as having the highest predicted binding affinities were all primates, including humans.
Other taxa with high predicted affinities included other primates, rodents, even-toed ungulates (namely, several species of cetaceans and deer), and anteaters.
Reindeer were the only domesticated species predicted to belong to either of these groups, but many common zoo animal species with threatened or worse IUCN risk status were identified as at risk.

> Many cases of reverse zoonotic, or anthroponotic, SARS-CoV-2 transmission have been reported.

Reviewer: ```suggestion
Considering the evidence generated by _in silico_ studies, it may not be surprising that many cases of reverse zoonotic, or anthroponotic, SARS-CoV-2 transmission have been reported.
```

Ferrets (_Mustela furo_) as well as cats and dogs were reported to be susceptible to SARS-CoV-2 in an experimental infection study [@doi:10.1111/xen.12621].
The earliest reported anthroponotic transmission events were observed in house pets, primarily cats (_Felis catus_) [@doi:10.3390/v13020185; @doi:10.1038/s41586-020-2334-5; @doi:10.3201/eid2707.204959].

> Similarly, cases of SARS-CoV-2 infection have been reported in dogs (_Canis familiaris_): two of fifteen dogs monitored for SARS-CoV-2 by the Hong Kong Agriculture, Fisheries and Conservation Department during the owners' quarantine in March 2020 were found to be positive for SARS-CoV-2 [@doi:10.1038/s41586-020-2334-5]. 

Reviewer: ```suggestion
Similarly, cases of SARS-CoV-2 infection have been reported in dogs (_Canis familiaris_): two of fifteen dogs monitored for SARS-CoV-2 by the Hong Kong Agriculture, Fisheries, and Conservation Department during the owners' quarantine in March 2020 were found to be positive for SARS-CoV-2 [@doi:10.1038/s41586-020-2334-5]. 
```

Reviewer: Apologies that change was to satisfy my inner comma goblin

This finding was supported by experimental evidence that SARS-CoV-2 cannot use mouse (_Mus musculus_) ACE2 for cell entry [@doi:10.1038/s41586-020-2012-7].
In fact, research using murine models to study SARS and COVID-19 therefore uses transgenic mice designed to be sensitive to the virus (as summarized in [@doi:10.1007/s11259-021-09787-2]).

Similarly, SARS-CoV-2 in livestock also raised concern because of the potential effect on food supply.
However, studies using _in vivo_ viral challenge reported that livestock species in general do not develop clinical manifestations of SARS-CoV-2 and do not shed infectious virus [@doi:10.1080/22221751.2021.2003724; @doi:10.1111/xen.12621].
_In vitro_ exposure to SARS-CoV-2 suggested that sheep (_Ovis aries_), but not cattle (_Bos taurus_), might be susceptible to infection, but _in vivo_ viral challenge suggested that sheep did not show notable susceptibility to infection [@doi:10.1080/22221751.2022.2037397].
Similarly, analyses of antibody response [@doi:10.3390/ani11071984] suggested that sheep exposed to a high level of human interaction did not appear to have developed infections.

> Following viral challenge of several species, including cattle, sheep, and horses (_Equus ferus caballus_) , none were found to shed culturable levels of virus [@doi:10.1080/22221751.2021.2003724].

Reviewer: ```suggestion
Following viral challenge of several species, including cattle, sheep, and horses (_Equus ferus caballus_), none were found to shed culturable levels of virus [@doi:10.1080/22221751.2021.2003724].
```

As a note, despite the low risk posed by livestock themselves, the working conditions of the meat industry itself were associated with a very high risk of SARS-CoV-2 infection for workers that did cause disruptions to food supply chains [@doi:10.1073/pnas.2010115117; @doi:10.3389/fvets.2020.585787].

> However, one of species of domesticated agricultural animal is severely affected by SARS-CoV-2: the mink (_Neovison vison_).

Reviewer: ```suggestion
However, one species of domesticated agricultural animal severely affected by SARS-CoV-2 was the mink (_Neovison vison_).
```

The fact that these variants appeared in mink populations before being observed in humans suggests that mink can indeed serve as a viral reservoir [@doi:10.3201/eid2702.203794].
Concerns about mink-to-human transmission led to the mass destruction of domesticated mink populations in Europe [@doi:10.1016/j.nmni.2020.100816; @doi:10.1126/science.368.6496.1169].
Introgression from fur farms into wild populations (i.e., feralization) may have also resulted in the spread of SARS-CoV-2 into wild mink populations [@doi:10.1002/fee.2344; @doi:10.3390/ani11051422].
Therefore, while the specific zoonotic origin of SARS-CoV-2 may still not be clear, the potential for the virus to take hold in species other than humans has been clearly demonstrated by the mink outbreak.

Finally, some species of zoo animals were also monitored to determine whether they were at risk.
Several species closely related to humans (i.e., the Great Apes) are threatened with extinction and had been identified through _in silico_ studies as likely to be susceptible to SARS-CoV-2 [@doi:10.1073/pnas.2010146117], and therefore the potential for a virus to target these close relatives presented a major concern.

> In early 2021, three gorillas at the San Diego Zoo Safari Park developed respiratory symptoms that were confirmed to be associated with SARS-CoV-2 [@url:https://www.aphis.usda.gov/aphis/newsroom/stakeholder-info/sa_by_date/sa-2021/sa-01/ca-gorillas-sars-cov-2].<!--I can't find which species!-->

Reviewer: Probably _Gorilla beringei beringei_ 

Here is my links to how I found out haha
https://stories.sandiegozoo.org/zoonooz/wonderful-winston/
https://stories.sandiegozoo.org/2021/01/25/gorillasrecovering/
https://www.microbe.tv/twiv/twiv-745/

Author: TWIV strikes again!

```suggestion
In early 2021, three gorillas (_Gorilla beringei beringei_) at the San Diego Zoo Safari Park developed respiratory symptoms that were confirmed to be associated with SARS-CoV-2 [@url:https://www.aphis.usda.gov/aphis/newsroom/stakeholder-info/sa_by_date/sa-2021/sa-01/ca-gorillas-sars-cov-2].
```

For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948], though select VOCs may present challenges. 
Further research is required to discern the clinical, prophylactic, and therapeutic consequences of these genetic SARS-CoV-2 variants as the pandemic evolves. 

> ### Question 6: What is Next?

Author: This is just a draft, probably needs some additions

The SARS-CoV-2 pandemic has presented many unprecedented scientific opportunities.
The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses, and the scientific community has continued to collect, analyze, and disseminate information about the SARS-CoV-2 virus and the associated illness, COVID-19 at previously unimaginable rates [@individual-methods].
The accessibility of genome sequencing technology has allowed for deep sequencing of the virus to establish a level of viral surveillance that had never before been achieved [@doi:10/d5mg; @doi:10.1038/s41592-022-01444-z; @doi:10.1016/j.lanepe.2021.100210].
The information obtained from genetic, bioinformatics, and evolutionary analysis has played a significant role in shaping the global pandemic response [@doi:10.1038/s41592-022-01444-z; @doi:10.1038/s41586-021-04188-6; @doi:10/hskp].

> <!-- Wastewater surveillance? Implications for preventing future pandemics? Vaccine response time?-->

Author: Maybe a good spot for the text @RLordan and @Sergey-Knyazev were thinking of adding?

Reviewer: Yes, apologies, bare with me will get to it tomorrow

Author: No pressure @RLordan! You said you were busy through Tuesday but I just thought it might fit there so I thought I'd make a note-- and of course if you'd rather put it somewhere else, that is totally fine as well!

Reviewer: @rando2 I have a few lines on this when this is merged :)
